string object_name
uint16 method
uint16 num_samples
uint16 num_images
float32 noise_level
---
float32 rec_rate
float32 sigma_pose
float32[] prob_dist

float32 sigma_filtered
float32[] filt_dist 